packages/store/internals/testing/src/fresh-platform.ts

```diff
@@ -30,11 +30,26 @@ function resetPlatformAfterBootstrapping() {
   createPlatform(TestBed);
 }
 
-export function freshPlatform(fn: Function): (...args: any[]) => any {
-  return async function testWithAFreshPlatform(this: any, ...args: any[]) {
+export function freshPlatform(fn: (done?: VoidFunction) => Promise<void>) {
+  let done: VoidFunction | null = null,
+    whenDoneIsCalledPromise: Promise<void> | null = null;
+
+  if (fn.length === 1) {
```

Comment:
`fn.length === 1` what does it mean? can we give a semantic meaning to the name of the variable?

Reply:
I'll add comments.

Reply:
Yes, please add a named variable. Possibly `hasDoneArgument`?